server/consumer.go

```diff
@@ -1703,7 +1703,7 @@ func (o *consumer) deleteNotActive() {
 			// Don't think this needs to be a monitored go routine.
 			go func() {
 				const (
-					startInterval = 30 * time.Second
+					startInterval = 5 * time.Second
```

Comment:
Why is this being changed in a PR about meta snapshots?

Reply:
Sending the snapshot was forcing the ghost consumers tests to pass for the wrong reason.

When we removed it, the test started to flake unless it was given over a minute to run. Reducing the initial time to clean up those ghost consumers de-flaked those tests without having to extend the test to such a long runtime.

Reply:
Yes but that can dramatically impact production systems with 10s or 100s of thousands of these. Hence the 30s vs shorter.

If need be make a var off of a const and set var to what you need in test and reset at the end of the test.

Reply:
OK will do tomorrow.

Reply:
In fact have just done now, as it was a quick change.

Reply:
Just removed the accidental `const ()` that was left over too.